demos/m5/overlays/dev/release.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Release
+metadata:
+  name: m5-release
+  namespace: demo
+spec:
+  applicationSnapshot: m5-snapshot
```

Comment:
I think this PR should add a snapshot (even if it is a dummy one like the params in the strategy)